.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -0,0 +1,812 @@
+          <li class="primary-link">
+            <a>About</a>
+            <div class="global-nav-dropdown">
+              <ul>
+                <li><a class="icon icon-location" href="/about/maps-directions/">Maps and Directions </a></li>
+                <li><a class="icon icon-california" href="/about/visit-chapman/">Visit Chapman</a></li>
+                <li><a class="icon icon-cu-monogram" href="/discover/index.html">Discover Chapman</a></li>
+                <li><a class="icon icon-office" href="/about/campus/">Our Campus</a></li>
+                <li><a class="icon icon-cu-monogram" href="/about/facts-history/">Facts and History</a></li>
+                <li><a class="icon icon-cu-window" href="/about/administration/">Administration</a></li>
+                <li><a class="icon icon-envelop" href="/about/contact-us.aspx">Contact Us</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="primary-link">
+            <a>Academics</a>
+            <div class="global-nav-dropdown">
+              <ul>
+                <li><a class="icon icon-graduation" href="/academics/degrees-and-programs.aspx">Degrees &amp; Programs</a></li>
+                <li><a class="icon icon-library2" href="/academics/schools-colleges.aspx">Schools &amp; Colleges</a></li>
+                <li><a class="icon icon-calendar4" href="/academics/academic-calendar.aspx">Academic Calendar</a></li>
+                <li><a class="icon icon-books" href="/academics/libraries/">Libraries</a></li>
+                <li><a class="icon icon-book2" href="/academics/course-catalogs/">Course Catalogs</a></li>
+                <li><a class="icon icon-earth" href="/international-studies/">International Study</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="primary-link">
+            <a>Admission</a>
+            <div class="global-nav-dropdown">
+              <ul>
+                <li><a class="icon icon-bookmark" href="/admission/undergraduate/">Undergraduate Admission</a></li>
+                <li><a class="icon icon-pencil5" href="/admission/undergraduate/applynow.aspx">Undergraduate Application</a> </li>
+                <li><a class="icon icon-bookmark2" href="/admission/graduate/">Graduate Admission</a></li>
+                <li><a class="icon icon-pencil5" href="/admission/graduate/applynow.aspx">Graduate Application</a></li>
+                <li><a class="icon icon-calculate2" href="/admission/undergraduate/afford.aspx">Affordability</a></li>
+                <li><a class="icon icon-calculate" href="/students/tuition-and-aid/financial-aid/net-cost-calculator/">Financial Aid Calculator</a></li>
+                <li><a class="icon icon-office" href="/admission/undergraduate/visit.aspx">Campus Tours</a></li>
+              </ul>
+            </div>
+          </li>
+          <li class="primary-link">
+            <a>Faculty</a>
+            <div class="global-nav-dropdown">
```

Comment:
Notice: next 2 sections are all Law links and window icons. Let me know if this should be amended. Does not necessarily have to be changed now. Can be change in "final touches" story.